Integrations/integration-MinervaLabsAntiEvasionPlatform.yml

```diff
@@ -0,0 +1,293 @@
+description: Minerva eliminates the endpoint security gap while empowering companies
+  to embrace technology fearlessly.
+detaileddescription: |-
+  Minerva Labs integration with Demisto.
+  The integration queries the Minerva server for new events, endpoint status, groups and able to create active response such as adding vaccinations and exclusions.
+configuration:
+- display: url
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: user
+  name: user
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: password
+  name: password
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: insecure
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+script:
+  script: |
+    # ''' IMPORTS '''
+    import json
+
+    import requests
+
+    # disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    # ''' GLOBALS '''
+    USERNAME = demisto.params().get('user')
```

Comment:
After changing the parameter type to credentials, you would need to use `demisto.params()['credentials'].identifier/password`